crates/oxc_parser/src/diagnostics.rs

```diff
@@ -217,8 +217,9 @@ pub fn spread_last_element(span: Span) -> OxcDiagnostic {
 }
 
 #[cold]
-pub fn binding_rest_element_trailing_comma(span: Span) -> OxcDiagnostic {
-    OxcDiagnostic::error("Unexpected trailing comma after rest element").with_label(span)
+pub fn rest_parameter_or_binding_pattern_may_not_have_trailing_comma(span: Span) -> OxcDiagnostic {
+    ts_error("1013", "A rest parameter or binding pattern may not have a trailing comma.")
```

Comment:
This is also a normal js error, so we shouldn't display this as ts specific (ts error code).